tools/deployment/package/docker-compose.yaml

```
@@ -140,3 +140,28 @@ services:
       "--concurrency", "${CLP_REDUCER_CONCURRENCY:-1}",
       "--upsert-interval", "${CLP_REDUCER_UPSERT_INTERVAL:-100}"
     ]
+
+  mcp-server:
+    <<: *service_defaults
+    hostname: "mcp_server"
+    environment:
+      CLP_LOGS_DIR: "/var/log/mcp-server"
+      PYTHONPATH: "/opt/clp/lib/python3/site-packages"
+    ports:
+      - "${CLP_MCP_HOST:-127.0.0.1}:${CLP_MCP_PORT:-8000}:8000"
+    volumes:
+      - "${CLP_MCP_LOGS_DIR_HOST:-./var/log/mcp-server}:/var/log/mcp-server"
+    healthcheck:
+      <<: *healthcheck_defaults
+      test: [
+        "CMD",
+        "curl",
+        "-f",
+        "http://${CLP_MCP_CONTAINER_HOST:-0.0.0.0}:${CLP_MCP_CONTAINER_PORT:-8000}/health"
+      ]
+    command: [
+      "python3", "-u",
+      "-m", "clp_mcp_server.clp_mcp_server",
+      "--host", "${CLP_MCP_CONTAINER_HOST:-0.0.0.0}",
+      "--port", "${CLP_MCP_CONTAINER_PORT:-8000}",
```

ditto. we can hard code this port. the port mapping in above `ports` specification is hardcoded anyways
```suggestion
      "--port", "${CLP_MCP_CONTAINER_PORT:-8000}",
```

---

didn't know we can hard code this, I will get rid of variable for CLP_MCP_CONTAINER_HOST and CLP_MCP_CONTAINER_PORT then.